src/utils/mesh.js

```diff
@@ -58,12 +61,12 @@ function getCSaaSMeshConfig (core, githubOrg, githubRepo) {
                 }
                 },
                 "responseConfig": {
-                    "headers": ["X-Magento-Cache-Id"]
+                "headers": ["X-Magento-Cache-Id"]
```

Comment:
weird removal of indention :D I assume this is lint?

Reply:
Ohh yeah, let me handle this. Definetly a weird lint fix. I have been trying `yarn lint --fix` to match your style.

Also I was editing manually, maybe my manual edits caused this. Ill fix it and the other ones.